PatientWebApplication/AppointmentMicroserviceApi/Model/Survey.cs

```diff
@@ -0,0 +1,118 @@
+﻿using AppointmentMicroserviceApi.Model;
+
+namespace AppointmentMicroserviceApi.Patient
+{
+    public class Survey : Entity
+    {
+        //for patients
+        public int patientId { get; set; }
+
+        //for doctors
+        public virtual DoctorAppointment appointment { get; set; }
+
+        public int appointmentId { get; set; }
+
+        public int doctorsProfessionalism { get; set; }
+        public int doctorsPoliteness { get; set; }
+
+        public int doctorsTechnicality { get; set; }
+
+        public int doctorsSkill { get; set; }
+
+        public int doctorsKnowledge { get; set; }
+
+        public int doctorsWorkingPace { get; set; }
+
+        //for medical staff
+        public int medicalStaffsProfessionalism { get; set; }
+        public int medicalStaffsPoliteness { get; set; }
+        public int medicalStaffsTechnicality { get; set; }
+        public int medicalStaffsSkill { get; set; }
+        public int medicalStaffsKnowledge { get; set; }
+        public int medicalStaffsWorkingPace { get; set; }
+
+        //for hospital
+
+        public int hospitalEnvironment { get; set; }
+
+        public int hospitalEquipment { get; set; }
+
+        public int hospitalHygiene { get; set; }
+        public int hospitalPrices { get; set; }
+        public int hospitalWaitingTime { get; set; }
```

Comment:
Change name of properties to start with upper letter. After remove 'this' in constructors.